components/clarinet-format/tests/golden-intended/clarity-bitcoin.clar

```diff
@@ -0,0 +1,1119 @@
+;; source: https://github.com/hirosystems/clarity-examples/blob/main/examples/clarity-bitcoin/contracts/clarity-bitcoin.clar
+;; @contract stateless contract to verify bitcoin transaction
+;; @version 5
+;; version 5 adds support for txid generation and improves security
+;; Error codes
+(define-constant ERR-OUT-OF-BOUNDS u1)
+(define-constant ERR-TOO-MANY-TXINS u2)
+(define-constant ERR-TOO-MANY-TXOUTS u3)
+(define-constant ERR-VARSLICE-TOO-LONG u4)
+(define-constant ERR-BAD-HEADER u5)
+(define-constant ERR-HEADER-HEIGHT-MISMATCH u6)
+(define-constant ERR-INVALID-MERKLE-PROOF u7)
+(define-constant ERR-PROOF-TOO-SHORT u8)
+(define-constant ERR-TOO-MANY-WITNESSES u9)
+(define-constant ERR-INVALID-COMMITMENT u10)
+(define-constant ERR-WITNESS-TX-NOT-IN-COMMITMENT u11)
+(define-constant ERR-NOT-SEGWIT-TRANSACTION u12)
+(define-constant ERR-LEFTOVER-DATA u13)
+;;
+;; Helper functions to parse bitcoin transactions
+;;
+;; Create a list with n elments `true`. n must be smaller than 9.
+(define-private (bool-list-of-len
+    (n uint)
+  )
+  (unwrap-panic (slice? (list true true true true true true true true) u0 n))
+)
+
+;; Reads the next two bytes from txbuff as a little-endian 16-bit integer, and updates the index.
+;; Returns (ok { uint16: uint, ctx: { txbuff: (buff 4096), index: uint } }) on success.
+;; Returns (err ERR-OUT-OF-BOUNDS) if we read past the end of txbuff
+(define-read-only (read-uint8
+    (ctx {
+      txbuff: (buff 4096),
+      index: uint,
+    })
+  )
+  (let (
+    (data (get txbuff ctx))
+    (base (get index ctx))
+  )
+    (ok
+      {
+        uint8: (buff-to-uint-le
+          (unwrap-panic
+            (as-max-len?
+              (unwrap! (slice? data base (+ base u1)) (err ERR-OUT-OF-BOUNDS))
+              u1
+            ))
+        ),
+        ctx: {
+          txbuff: data,
+          index: (+ u1 base),
+        },
+      })
```

Comment:
The opening `(` has one less indentation level.
Although it would work to have both on the same